exp-player/addon/components/exp-card-sort/component.js

```diff
@@ -315,46 +326,8 @@ export default ExpFrameBaseComponent.extend({
           responses: {
               type: 'object',
               properties: {
-                  cardSort1: {
-                      uncharacteristic: {
-                          type: 'array'
-                      },
-                      neutral: {
-                          type: 'array'
-                      },
-                      characteristic: {
-                          type: 'array'
-                      }
-
-                  },
-                  cardSort2: {
-                      extremelyUnchar: {
-                          type: 'array'
-                      },
-                      quiteUnchar: {
-                          type: 'array'
-                      },
-                      fairlyUnchar: {
-                          type: 'array'
-                      },
-                      somewhatUnchar: {
-                          type: 'array'
-                      },
-                      neutral: {
-                          type: 'array'
-                      },
-                      somewhatChar: {
-                          type: 'array'
-                      },
-                      fairlyChar: {
-                          type: 'array'
-                      },
-                      quiteChar: {
-                          type: 'array'
-                      },
-                      extremelyChar: {
-                          type: 'array'
-                      }
+                  responses: {
+                      type: 'object'
```

Comment:
Is the server in any way responsible for validating that all questions are filled out? If we totally control the front-end code, and how users interact with the survey, this may be acceptable- just so long as the decision to remove backend validation is a conscious and active design choice.